docs/install.md

```diff
@@ -74,3 +87,41 @@ The different supported modes can be set by using the `hook-mode` option:
 - `podinformer`: Use a Kubernetes controller to get information about new pods. This option is racy and the first events produced by a container could be lost. This mode is selected when `auto` is used and the above modes are not available.
 - `ldpreload`: Adds an entry in `/etc/ld.so.preload` to call a custom shared library that looks for `runc` calls and dynamically adds the needed OCI hooks to the cointainer `config.json` specification. This feature only works when runc is used. Since this feature is highly experimental, it'll not be considered when `auto` is used.
 - `nri`: Use the [Node Resource Interface](https://github.com/containerd/nri). It requires containerd v1.5 and it's not considered when `auto` is used.
+
+### Specific Information for Different Platforms
+
+This section explains the additional steps that are required to run Inspektor
+Gadget in some platforms.
+
+#### Minikube
+
+You should create the minikube cluster in different ways according to the gadget
+you want to use.
+
+##### Using a VM driver
+
+This option uses a VM driver (like Virtualbox or kvm2). It's compatible with all
+the gadgets that don't require BTF support. This option uses a custom minikube
+image that contains a more recent kernel version (5.4.40) and some features
+enabled to make eBPF programs work there. More details are available
+[here](https://github.com/kinvolk/cloud-native-bpf-workshop/blob/master/minikube.md#our-branch).
+
+The traceloop and network policy advisor gadgets only work with this driver.
+
+```
+$ wget https://cloud-native-bpf-workshop-public.s3.eu-central-1.amazonaws.com/minikube.iso
+$ minikube start --driver=kvm2 --iso-url=file://$(pwd)/minikube.iso
+
+# Deploy Inspektor Gadget in the cluster as described above
+```
+
+##### Using the docker driver
+
+This option works for tools that require BTF support. You must have BTF support
```

Comment:
What happens if there's no BTF support but the person uses the image from above?

Reply:
It doesn't work because that image have no BTF support.

Reply:
Ok, I think you mean that the BTF tools don't work, but the other tools do work (as with the image above)? If so, I think I would split the content differently. Instead of by driver, split it by "Using a pre-provided image", which works with both VM and docker driver, and "Using upstream's image with the docker driver on a host with BTF enabled". What do you think?

Reply:
Is it possible to use the VM image with the Docker driver? :thinking:  /cc @alban

Reply:
No because the Docker driver uses the `gcr.io/k8s-minikube/kicbase` Docker image as the base for the host (built with `deploy/kicbase/Dockerfile`), but the VM image builds everything with Buildroot.